src/seedu/addressbook/data/person/Name.java

```diff
@@ -43,7 +43,12 @@ public static boolean isValidName(String test) {
     public List<String> getWordsInName() {
         return Arrays.asList(fullName.split("\\s+"));
     }
-
+    
+    public int compareTo(Name target) {
+        int compare = this.fullName.compareTo(target.fullName);
+        return(compare);
```

Comment:
I think it would be better to return the variable `compare` instead of initializing it beforehand, since it is redundant to do so.

Reply:
Ahh that is true. Updated my code to reflect that change.